sentry-ruby/lib/sentry/transaction.rb

```diff
@@ -108,6 +111,8 @@ def set_initial_sample_decision(sampling_context:, configuration: Sentry.configu
     end
 
     def finish(hub: nil)
```

Comment:
I think callers should never pass a different hub here than the one used to start the transaction, but I'm okay if it is not removed from the API (I'm aware Python has it for historical reasons, committed 2 years ago...)